mne_bids/datasets.py

```diff
@@ -38,3 +41,67 @@ def fetch_faces_data(data_path, repo, subject_ids):
             tf.extractall(path=data_path)
             shutil.move(op.join(data_path, 'ds117'), target_dir)
             os.remove(tar_fname)
+
+
+def download_matchingpennies_subj(url_dict=None, directory='.'):
```

Comment:
couple of questions

1. this dataset is already in BIDS format?
2. how big is it?
3. is it going to be used in an example somehow?

Reply:
1. Yes, I added it to offer a quick and easy access to straight forward EEG files (only 2 markers corresponding to two conditions in a single task ... 10 electrodes EEG, that's it for the data)
2. The whole dataset something between 5 and 6 GB. For testing, I just download a channels file which is several kB - for actually doing something with it, one subj suffices, which is <1GB
3. Yes, I'll use it in the EEG example later on

Reply:
okay looking forward to the example!